src/components/Editor.jsx

```diff
@@ -72,6 +72,12 @@ class Editor extends React.Component {
     } = this.state
 
     const codeMirrorInst = this.codeMirror.getCodeMirror();
+
+    // If this is the first time through, value may be set without Codemirror
+    // knowing about it. Push the value to codemirror to rectify
+    if (codeMirrorInst.getValue() === '' && value !== '') {
+      codeMirrorInst.setValue(value);
+    }
```

Comment:
One thing that I don't like is the need to do these comparisons every time `componentDidUpdate()` is called. I do suggest, since we should only need to do this once, is to use lodash's [`once()`](https://lodash.com/docs/4.17.4#once) function, which should eliminate the need for those conditionals.